mm2src/coins/qrc20/qrc20_tests.rs

```diff
@@ -150,7 +152,7 @@ fn test_validate_maker_payment() {
 
     assert_eq!(
         *coin.utxo.derivation_method.unwrap_single_addr(),
-        "qUX9FGHubczidVjWPCUWuwCUJWpkAtGCgf".into()
+        Address::from_legacyaddress("qUX9FGHubczidVjWPCUWuwCUJWpkAtGCgf", 120, 0, 50, 0).unwrap()
```

Comment:
I think you can use `coin.as_ref().conf.pub_addr_prefix/...` here and other similar lines. Also, QTUM regtest P2SH prefix is 110.

Reply:
I tried as_ref but I guess deref here is okay, the alternative is to use '&Address::from_legacyaddress(..).
Also instead of using consts I changed it to coin.as_ref().conf.address_prefixes